pages/gallery/gallery.css

```diff
@@ -0,0 +1,185 @@
+* {
+    margin: 0;
+    padding: 0;
+    font-size: 100%;
+    box-sizing: border-box;
+    font-family: 'Open Sans', sans-serif;
+}
+
+div{
+    width: 100%;
+    height: 100%;
+    text-align: center;
+    padding: 5px;
+    border: 10px solid darkseagreen;
+    background-color: mintcream;
+}
+
+img{
+    object-fit: cover;
+    width: 100%;
+    height: 95%;
+}
+
+/* Naviagation Panel */
+nav,
+ul {
+    list-style: none;
+    width: 100%;
+}
+
+.container {
+    width: 100%;
+    height: 100%;
+    padding: 10px 10px;
+    background-color: lightyellow;
+}
+
+.container li.menu-item {
+    font-size: 30px;
+    box-sizing: border-box;
+}
+
+.container nav,
+.container ul {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    flex: 1;
+    width: 100%;
+    list-style: none;
+}
+
+.container li a {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    padding: 5px 5px;
+    height: fit-content;
+    color: forestgreen;
+    background-color: burlywood;
+    border: 10px mintcream dotted;
+    text-shadow: 2px 2px lavenderblush;
+    text-decoration: none;
+    
+
+}
+
+.container li a:hover {
+    background-color: hotpink;
+}
+
+.container li {
+    margin: 50px;
+}
+
+
```

Comment:
Small style/organization note that applies to all the CSS files: to be consistent, I would remove the blank lines inside rule sets and keep a single blank line between each rule set to separate them.